site/i18n/en.json

```diff
@@ -44,6 +44,9 @@
   "menu_api": {
     "other": "API"
   },
+  "index_campaign_by": {
+    "other": "A campaign by"
```

Comment:
I would propose "initiative" or "activity". A campaign, in my understanding, is time-limited.

We have been discussing that in FSFE for a long time, but in REUSE we've been speaking about "initiative" since 2019 at least.

Reply:
Agreed